sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/key/RsaPublicKeyLoader.java

```diff
@@ -1,6 +1,6 @@
 package org.sdase.commons.server.auth.key;
 
-import java.security.interfaces.RSAPublicKey;
+import com.google.common.collect.Lists;
```

Comment:
I'd prefer a simple `new ArrayList(...)` instead of the dependency to the google lib.